src/rpc/budget.cpp

```diff
@@ -453,13 +455,10 @@ UniValue getbudgetprojection(const JSONRPCRequest& request)
             "    \"Yeas\": n,                    (numeric) Number of yea votes\n"
             "    \"Nays\": n,                    (numeric) Number of nay votes\n"
             "    \"Abstains\": n,                (numeric) Number of abstains\n"
-            "    \"TotalPayment\": xxx.xxx,      (numeric) Total payment amount\n"
-            "    \"MonthlyPayment\": xxx.xxx,    (numeric) Monthly payment amount\n"
-            "    \"IsEstablished\": true|false,  (boolean) Established (true) or (false)\n"
-            "    \"IsValid\": true|false,        (boolean) Valid (true) or Invalid (false)\n"
-            "    \"IsInvalidReason\": \"xxxx\",      (string) Error message, if any\n"
```

Comment:
Yeah, we cannot simply remove them from the help, since they  are returned from `budgetToJSON`.
We either use a different ToJSON function (but I don't think it's worth) or we need to add a meaningful description.

Reply:
alright will adjust now

Reply:
yeah. Shouldn't had been removed without remove them from the output. Can add a boolean to `budgetToJSON` to add them or not as well. The function is very simple.

Reply:
I think it would be overkill, but no big deal.
Anyway we still need to find a good description for "IsEstablished", in the rpc help (at least for `getbudgetinfo`), which was the original goal of the PR :)

Reply:
I wouldn't label as an overkill a 3 lines change :p.
but anyway, based on discord's text, the initial goal of the PR was to improve `getbudgetprojection` help description as liquid was asking about the requirements for proposals to be returned by this command. (Thus why wrote the two probable confusion points [here](https://github.com/PIVX-Project/PIVX/pull/2231#issuecomment-791398737)).

either way, all good. not a big deal.